website/pages/style.css

```diff
@@ -20,6 +20,7 @@
 @import '~@hashicorp/react-docs-page/style.css';
 @import '~@hashicorp/react-docs-sidenav/style.css';
 @import '~@hashicorp/react-enterprise-alert/style.css';
+@import '~@hashicorp/react-featured-slider/style.css';
```

Comment:
`featured-slider` isn't on CSS modules yet, so we still need to import styles globally.